app/src/main/java/com/cpr3663/cpr_scouting_app/PreMatch.java

```diff
@@ -113,31 +137,14 @@ protected void onCreate(Bundle savedInstanceState) {
             // If we got match data, get the list of teams from it
             if (Match != null) {
                 ArrayList<String> teamsInMatch = Match.getListOfTeams();
-                boolean found = false;
```

Comment:
Why is this removed?  Two things: if you override and add a team, this removed code handles that.  without it, you won't get the right one.  Second, setSelection() inputs the POSITION, not the text.  so setSelection(TeamToScout) doesn't work.